tests/unit/test_file.py

```diff
@@ -563,9 +563,8 @@ def test_simple(self):
         with NWBHDF5IO(self.path, 'w') as io:
             io.write(nwbfile, cache_spec=False)
 
-        with self.assertWarnsWith(UserWarning, "No cached namespaces found in %s" % self.path):
-            with NWBHDF5IO(self.path, 'r', load_namespaces=True) as reader:
-                nwbfile = reader.read()
+        with NWBHDF5IO(self.path, 'r') as reader:
+            nwbfile = reader.read()
```

Comment:
What is this case testing, it does not seem to assert anything? Has the warning been removed when no namespace has been cached?